android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayout.java

```diff
@@ -192,6 +192,7 @@ void onNativeLibraryReady() {
 
       SharedPreferences sharedPreferences = ContextUtils.getAppSharedPreferences();
       if (ChromeFeatureList.isEnabled(BraveFeatureList.BRAVE_REWARDS)
+              && !BravePrefServiceBridge.getInstance().getSafetynetCheckFailed()
```

Comment:
if this is a bug fix for 81 it should go in a separate PR so it can get proper QA, etc.. and uplift

Reply:
cc @bsclifton

Reply:
cc: @samartnik

Reply:
Definitely we'll want an issue that we can link https://github.com/brave/brave-core/commit/5762ffebca9223407b53fb96b8a4c8172e76cd08 against. Ideal situation would be separate PR. Can you create an issue and consider pulling out of this PR (in favor of a separate PR) @samartnik? 😄

Reply:
PR created https://github.com/brave/brave-core/pull/5612
Commit reverted https://github.com/brave/brave-core/commit/2ec12f4bd34a8504f7a2029fbe31a5ad804e29a7